src/renderer/tvgTrimPath.cpp

```diff
@@ -205,8 +206,8 @@ static void _trimPath(const PathCommand* inCmds, uint32_t inCmdsCnt, const Point
     bool start = true;
     for (uint32_t i = 0; i < inCmdsCnt; ++i) {
         auto dLen = _length();
-        //skip the segment
-        if (len + dLen <= trimStart || len > trimEnd) {}
+        //skip the segment (also skip segments with len < EPSILON
```

Comment:
Please add a descriptive comment about epsilon at the code.
```
Due to the finite precision of Bezier curve subdivision and length calculation (1e-2), 
trimming may produce very short segments that would effectively have zero length with higher computational accuracy.
This helps to prevent unwanted artifacts during dashing.
```

Reply:
Didn't mention dashing, as it's not related.
as I wrote in the previous commit msg a similar approach is implemented in the sw engine for dashing (https://github.com/thorvg/thorvg/pull/1932), but these pr doesn't affect anything related to dashing.